src/Compiler/Scheme/Racket.idr

```diff
@@ -331,6 +331,7 @@ getFgnCall appdir (n, fc, d) = schFgnDef appdir fc n d
 startRacket : String -> String -> String -> String
 startRacket racket appdir target = unlines
     [ "#!/bin/sh"
+    , "# this file is @" ++ "generated by Idris2 compiler, Racket backend"
```

Comment:
Why isn't this a single string? Lots of weird code with no comment explaining the
rational for it in this PR.

Reply:
Drop the `@`, surely the note is plenty... everything in `build/` is generated.
`"/* automatically generated using the Idris2 C Backend */"` is used in RefC, so follow that?

IdrisPaths is the exception, since it's generated into `src/`. How about `echo '-- @'generated ...` or `echo -- @\generated ...`?

Reply:
`@generated` is not exactly standard, but there's a little tendency to use it. For example, it is understood by the Phabricator tool, and it is generated by Rust Cargo into generated files.

Advantage of `@generated` against any other string, is that it is machine/script friendly. E. g. `grep @generated` and you'll file all generated files.

So, I'd prefer sticking to `@generated`. Or you still think it's better to drop at-symbol?

Reply:
does Idris2 support hex escapes? maybe just put the `@` in escaped form in there.

Reply:
It's the concatenation we are objecting to. The point about `grep` is valid, but the inverse is equally true (i.e. searching for the generating code). So any form of obfuscation is a no-go.

The `@` could be helpful, but is probably better reserved for its current use (IMO).
You could argue that `bootstrap/idris2_app/idris2.{rkt,ss}` ought to have `@generated`, since they aren't `.gitignore`d.

Reply:
wouldn't simply putting `@` in as a hex escape solve the first point, e.g.
`"# this file is \x40generated by Idris2 compiler, Racket backend"`

Reply:
That's replacing one form of obfuscation with another, a side step rather than a solution. You'd need somehow know to `grep \\\\x40generated` to (maybe) find the source of a file.
Keep it simple, `"# automatically generated using the Idris2 Racket Backend"` or `"# @generated"` is consistent with what's existing.

Reply:
Alright if it helps with the tooling then let's go with it.